packages/client/hmi-client/src/components/workflow/ops/funman/tera-funman-drilldown.vue

```diff
@@ -920,6 +971,17 @@ function updateSelectedParameters(event) {
 	updateParameterCharts(); // Rerender when we remove/add charts since they are all contained in the same visualization
 }
 
+function updateSelectedObservables(event) {
```

Comment:
I think you can just use `updateChartSettings` method in `useChartSettings.ts` instead.